extensions/wikia/Email/Controller/FounderController.class.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\Check;
+use Email\EmailController;
+use Email\Fatal;
+
+abstract class FounderController extends EmailController {
```

Comment:
So this class is missing tracking info, which means all of these emails will use the default category of "UserMailer".

The way we have been adding tracking categories is via a class constant like [this](https://github.com/Wikia/app/blob/dev/extensions/wikia/Email/Controller/ForumController.class.php#L11). The problem for this ticket though is that there looks like there's a lot more logic in place for the tracking category (FounderEmailsEditAnonEN versus FounderEmailsEditAnonINT versus FounderEmailsEditUserEN).

Not sure what the best approach would be, maybe change the constants to a method which can be overridden? Or we could check with Cristina to see if these fine-grained categories are even useful any more and, if not, just give all of these a category of "FounderEmail."